src/rules/no-alias-methods.ts

```diff
@@ -1,19 +1,25 @@
-import { expectCaseWithParent, getDocsUrl, method } from './util';
+import { createRule, isExpectCall, parseExpectCall } from './tsUtils';
```

Comment:
I'll probably use this to land `parseExpectCall`, followed by `no-truthy-falsy`, and then the other rules can have their own PRs 🤷‍♂